model/service/AbstractIrregularityReport.php

```diff
@@ -104,18 +105,21 @@ public function __invoke($params)
 
         return $filePrefix === false
             ? Report::createFailure(__('Unable to create irregularities export for %s', $delivery->getLabel()))
-            : Report::createSuccess(__('Irregularities for "%s" successfully exported', $delivery->getLabel()), $filePrefix);
+            : Report::createSuccess(
+                __('Irregularities for "%s" successfully exported', $delivery->getLabel()),
```

Comment:
I am not sure we can line break the arguments

Reply:
These line breaks are added to `Report::createSuccess`, not to `__(...)`